versioned/persist/adapter/src/test/java/org/projectnessie/versioned/persist/adapter/spi/TestFetchValuesUsingOpenAddressing.java

```diff
@@ -279,20 +347,15 @@ private static String intToPaddedHexString(int i) {
     return String.format("%02x", i);
   }
 
-  /**
-   * Builds a fake {@linkplain KeyListEntity}.
-   *
-   * <p>All keys appearing in this entity have a custom hashCode() that returns {@linkplain
-   * Integer#MAX_VALUE}.
-   */
-  private static KeyListEntity getKeyListEntity(int segmentIndex, int entryCount) {
-    String prefix = "seg" + intToPaddedHexString(segmentIndex) + "_";
-
+  /** Builds a fake {@linkplain KeyListEntity}. */
+  private static KeyListEntity getKeyListEntity(
+      int startingBucket, int segmentIndex, int entryCount, Function<Integer, Key> bucketToKey) {
     ImmutableKeyList.Builder listBuilder = ImmutableKeyList.builder();
 
+    int bucket = startingBucket;
     for (int i = 0; i < entryCount; i++) {
-      String keyString = prefix + i;
-      listBuilder.addKeys(getKeyListEntry(keyString));
+      listBuilder.addKeys(getKeyListEntry(bucketToKey.apply(bucket)));
+      bucket++;
```

Comment:
Super-nit: move into the for-loop

Reply:
done (I do not even recall when I used multiple variables in a `for` loop in java last time :sweat_smile: )